transform/nijz_weekly.py

```diff
@@ -167,7 +167,8 @@
     'week.from.kaz', 'week.from.xkx', 'week.from.hun', 'week.from.mkd', 'week.from.mlt', 'week.from.mar', 'week.from.fsm', 'week.from.deu',
     'week.from.pak', 'week.from.pol', 'week.from.rou', 'week.from.rus', 'week.from.svk', 'week.from.srb', 'week.from.esp', 'week.from.swe',
     'week.from.che', 'week.from.tur', 'week.from.ukr', 'week.from.gbr', 'week.from.usa', 'week.from.are', 'week.from.nld', 'week.from.prt', 
-    'week.from.lva', 'week.from.alb', 'week.from.cub', 'week.from.mli'
+    'week.from.lva', 'week.from.alb', 'week.from.cub', 'week.from.mli', 'week.from.egy', 'week.from.tza', 'week.from.fin', 'week.from.nam',
+    'week.from.kaz', 'week.from.irn', 'week.from.jib', 'week.from.mdv', 'week.from.qat'
```

Comment:
Countries `kaz` and `irn` are already listed above.

```suggestion
    'week.from.jib', 'week.from.mdv', 'week.from.qat'
```